tests/src/integration/common/wskdeploy.go

```diff
@@ -47,17 +47,17 @@ func (wskdeploy *Wskdeploy) RunCommand(s ...string) ([]byte, error) {
 }
 
 func (wskdeploy *Wskdeploy) Deploy(manifestPath string, deploymentPath string) ([]byte, error) {
-	return wskdeploy.RunCommand("-m", manifestPath, "-d", deploymentPath)
+	return wskdeploy.RunCommand("-i ", "false", "-m", manifestPath, "-d", deploymentPath)
```

Comment:
@pritidesai ,thanks for your review, yes, I think we need to change the command to
wskdeploy -i -m
Anyway, Vincent's latest PR will fix this issue I think.